src/main/java/com/suse/saltstack/netapi/datatypes/cherrypy/Applications.java

```diff
@@ -0,0 +1,117 @@
+package com.suse.saltstack.netapi.datatypes.cherrypy;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class Applications {
+
+    @SerializedName("Uptime")
+    private double uptime;
+
+    @SerializedName("Bytes Read/Second")
+    private double readsPerSecond;
+
+    @SerializedName("Current Time")
+    private Date currentTime;
+
+    @SerializedName("Server Version")
+    private String serverVersion;
+
+    @SerializedName("Total Time")
+    private double totalTime;
+
+    @SerializedName("Enabled")
+    private boolean enabled;
+
+    @SerializedName("Start Time")
+    private Date startTime;
+
+    @SerializedName("Bytes Written/Second")
+    private double writesPerSecond;
+
+    @SerializedName("Total Bytes Read")
+    private int totalBytesRead;
+
+    @SerializedName("Current Requests")
+    private int currentRequests;
+
+    @SerializedName("Total Requests")
+    private int totalRequests;
+
+    //TODO: implement requests
+    //@SerializedName("Requests")
+    //private Object requests;
```

Comment:
Maybe it's just a personal habit but I really dislike TODO comments and, even more, committed but commented out code.

For TODOs we have GitHub issues, which are harder to forget about and easier for others to pick up. Project management does not work well in source code.

For disabled code we have git - there's no need to keep unused, untested an probably not working code around.

So please remove these lines or consider handling Requests properly.